kernelci.org/content/en/docs/instances/local.md

```diff
@@ -0,0 +1,243 @@
+
+This section describes how to set up a KernelCI instance which is suitable for development. It's not meant to mimic the production-like setup on your local machine, but rather give directions on how to setup a minimal set of KernelCI components to simplify the process of development and testing new features.
+
+## Minimal set of KernelCI components
+
+The minimal setup for KernelCI consists of:
+
+- kernelci-backend
+- storage
+- kernelci-core
+
+This is sufficient for building the kernel, running tests and pushing build and test information as well as build artifacts to the backend and storage respectively.
+
+> # Note
+> These instructions are not suitable and should not be used to setup a production environment. They purposely neglect certain aspects of performance and security for the sake of simplicity.
+
+## Prerequisites
+
+Currently, the best option is to create a separate Virtual Machine that will run `kernelci-backend` and `storage`
+
+It's assumed in this document that you're running a following setup:
+
+- Virtual Machine running Debian Buster accessible via IP and domain name e.g. _kci-vm_. The following requirements need to be met:
+
+   - The VM must be reachable with a domain name from the host used for installation.
+   - SSH access is provided with key authorization
+
+> You can use DNS to access the VM with its domain name or simply put an entry to the `/etc/hosts` file.
+
+- Host machine which will be used to connect to the VM
+   It needs to meet following requirements:
+    - Installed apps:
+      - _git_
+      - _ansible_
+      - _Python 2.7_
+      - _Python 3.6+_
+
+## Deploy KernelCI backend and storage
+
+> # Note
+> It is assumed in this section that your kernelci VM is available from your host with the hostname `kci-vm`
+
+- Make sure _ansible_ is installed on the host machine.
+- Clone `kernelci-backend-config` repository
+
+    ```bash
+    git clone git@github.com:kernelci/kernelci-backend-config.git
+    ```
+
+- Configure ansible to be used with your VM
+
+    ```bash
+    cd kernelci-backend-config
+    ```
+
+    - Modify `[local]` section of the `hosts` file to match your configuration
+
+    ```ini
+    [local]
+    kci-vm ansible_ssh_user=kci
+    ```
+
+    - Create `host_vars/kci-vm`
+
+    ```bash
+    touch host_vars/kci-vm
+    ```
+
+    - Edit `host_vars/kci-vm` with your favorite text editor and put content there:
+
+    ```
+    hostname: kci-vm 
+    role: production
+    certname: kci-vm
+    storage_certname: kci-vm
+    kci_storage_fqdn: kci-vm
+    become_method: su
+    ```
+
+    - Create `dev.yml` playbook file with the following content:
+
+    ```
+    - hosts: kci-vm
+        become: yes
+        become_method: su
+        become_user: root
+        gather_facts: yes
+        roles:
+            - common
+            - install-deps
+            - install-app
+            - init-conf
+            - kernelci-storage
+
+    ```
+
+- Prepare secrets
+      - Copy `secrets.yml` template
```

Comment:
There seems to be a formatting problem here, the `- Copy` appears on the same line as `- Prepare secrets`